components/footer/Footer.tsx

```diff
@@ -202,9 +243,23 @@ export default function Footer() {
         <hr className="my-6 border-gray-200 dark:border-gray-700 sm:mx-auto lg:my-8" />
 
         <div className="sm:flex sm:items-center sm:justify-between">
-          <span className="mb-4 block text-sm text-gray-500 dark:text-gray-400 sm:mb-0 sm:text-center">
-            © 2021-present EternalCodeTeam. All rights reserved.
-          </span>
+          <div className="mb-4 sm:mb-0">
+            <span className="block text-sm text-gray-500 dark:text-gray-400 sm:text-center">
+              © 2021-present EternalCodeTeam. All rights reserved.
```

Comment:
To better match other formatting used in footer

```suggestion
              © 2021-present EternalCodeTeam | All rights reserved |
```